apps/server/src/adapters/mx.test.ts

```diff
@@ -468,13 +469,12 @@ describe('mx provider', () => {
 
       it('returns the member from aggregateMember if extended history is not supported', async () => {
         server.use(
-          http.post(EXTEND_HISTORY_PATH, () =>
-            HttpResponse.json({
-              error: {
-                message: EXTENDED_HISTORY_NOT_SUPPORTED_MSG
-              }
-            })
-          )
+          http.post(EXTEND_HISTORY_PATH, () => {
+            return HttpResponse.json(
+              { error: { message: EXTENDED_HISTORY_NOT_SUPPORTED_MSG } },
+              { status: 400 }
```

Comment:
Without a non-200 status, the code won't `catch`, which is what is necessary with this update to using `mx-platform-node`.